src/vis/vis.js

```diff
@@ -1152,6 +1152,32 @@ var Vis = cdb.core.View.extend({
         }
       });
     }
+
+    var self = this; 
+
+    if (layerView.tooltip){
+      
```

Comment:
Please, check the style of the code. There are several empty lines that should be removed (have a look to our JavaScript style guide wiki page)